api/v1beta1/kafkacluster_types.go

```diff
@@ -360,6 +362,14 @@ func (c ExternalListenerConfig) GetAnyCastPort() int32 {
 	return *c.AnyCastPort
 }
 
+func (c ExternalListenerConfig) GetBrokerPort(brokerId int32) int32 {
+	if c.ExternalStartingPort == -1 {
+		return *c.AnyCastPort
```

Comment:
this reads a bit confusing - can you add a comment here exlaining the logic?

Reply:
Also the CRD should contain some details on this `-1` value impact